src/block/utils/Copper.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ *
+ *  ____            _        _   __  __ _                  __  __ ____
+ * |  _ \ ___   ___| | _____| |_|  \/  (_)_ __   ___      |  \/  |  _ \
+ * | |_) / _ \ / __| |/ / _ \ __| |\/| | | '_ \ / _ \_____| |\/| | |_) |
+ * |  __/ (_) | (__|   <  __/ |_| |  | | | | | |  __/_____| |  | |  __/
+ * |_|   \___/ \___|_|\_\___|\__|_|  |_|_|_| |_|\___|     |_|  |_|_|
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * @author PocketMine Team
+ * @link http://www.pocketmine.net/
+ *
+ *
+ */
+
+declare(strict_types=1);
+
+namespace pocketmine\block\utils;
+
+/**
+ * Represents copper blocks that have oxidized and waxed variations.
+ */
+interface Copper{
```

Comment:
This should follow the existing interface naming scheme seen by other common trait/interface pairs in pocketmine.

Reply:
The problem is that it’s difficult to find a good naming. I can suggest the following names:
- CopperInterface
- ICopper
- CopperBase
- CopperBlock
- Cupreous
- OxidizableBlock
- OxidizableCopper

Reply:
I would prefer `ICopper` or `CopperBase`

Reply:
```ICopper``` is great

Reply:
> I would prefer `ICopper` or `CopperBase`

I think `ICopper` would be a good name because `CopperBase` sounds more like an abstract class rather than interface